MASFoundation/Classes/MAS.h

```diff
@@ -159,6 +159,26 @@
 
 
 
+/**
+*  Sets BOOL indicator of  enforcing SSL Pinning i.e. Evaluate Server Trust with pinned Server Certificates.
+*  By default, SSL Pinning is disabled.
+*
+*  @param enable BOOL value of indicating whether SSL Pinning should be enabled or not.
+*/
++ (void)enableSSLPinning:(BOOL)enable;
```

Comment:
Assuming that the internal property name is sslPinningEnabled,
the proper method could be: setSSLPinningEnabled(boolean).

But if you choose enableSSLPinning, avoid accepting the argument. Of course, to disable, we might require a separate method, i.e., disableSSLPinning.

**I recommend you to go by setter/getter terminology. I already see setters for other boolean properties.**

Reply:
Done.